mono/metadata/profiler.c

```diff
@@ -743,6 +789,38 @@ mono_profiler_class_loaded (MonoClass *klass, int result)
 	}
 }
 
+void
+mono_profiler_static_field (MonoClassField *field)
+{
+	for (ProfilerDesc *prof = prof_list; prof; prof = prof->next)
+		if ((prof->events & MONO_PROFILE_FIELD_EVENTS) && prof->static_field)
+			prof->static_field (prof->profiler, field);
+}
+
+void
+mono_profiler_static_fields_allocated (MonoClass *klass)
+{
+	for (ProfilerDesc *prof = prof_list; prof; prof = prof->next)
+		if ((prof->events & MONO_PROFILE_FIELD_EVENTS) && prof->static_fields_allocated)
+			prof->static_fields_allocated (prof->profiler, klass);
+}
+
+void
+mono_profiler_special_static_area_allocated (MonoObject *owner, gboolean thread_local, gsize owner_id, int index, void **addr, gsize size)
```

Comment:
We don't need it in the log profiler. I just figured I'd pass it in case some other profilers might find it useful.